jobserver/models/reports.py

```diff
@@ -74,9 +74,22 @@ class Meta:
     def __str__(self):
         return self.title
 
+    def get_absolute_url(self):
+        if hasattr(self, "analysis_request"):
+            return self.analysis_request.get_absolute_url()
+        else:
+            # we only have reports tied to analyses currently but we expect
+            # that to change in the future so rather than error out send the
+            # user _somewhere_.
+            return "/"
+
     def get_staff_url(self):
         return reverse("staff:report-detail", kwargs={"pk": self.pk})
 
+    @property
+    def is_draft(self):
```

Comment:
Would `is_released` be more consistent terminology or would it just lead to confusion?

Reply:
Confusion I think.  I see where you're coming from but releasing is specific to outputs and is the act of uploading filies from an L4 to job-server, where their contents can be published (and can thus be in a draft state!).